### PR TITLE
dunst: update to 1.6.1

### PR DESCRIPTION
**Pull request by biopsin**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

Drop make_build_target defaults to all
Add wayland support (default force_wayland = false)

### CODE REVIEW

**Comment by biopsin** [comment]
wtf

**Comment by FollieHiyuki** [comment]
Don't do `git merge` :) Do `git pull --rebase` from upstream branch instead

**Comment by FollieHiyuki** [comment]
Anyway `dunst` natively runs on Wayland is a huge deal for me. I was kinda waiting for this. 
Will try it out when I finish digging the doc and updating my config.

**Comment by biopsin** [comment]
@FollieHiyuki You should diff your dunstrc with the one in /etc/dunst as there are some changes.
Any feedback is appriciated, as I don't run wayland.

**Comment by FollieHiyuki** [comment]
Seems to work natively on `sway` for me. `xeyes` doesn't see the notifications.
The new progress bar feature works ok too.
I only use notification for basic backlight / volume control, so my testing ability is limited :)

File: srcpkgs/dunst/template [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-version=1.5.0
+version=1.6.1
 revision=1
 build_style=gnu-makefile
 make_use_env=yes
-make_build_target="all dunstify"
 hostmakedepends="perl pkg-config"
 makedepends="gtk+-devel libXScrnSaver-devel libXinerama-devel libXrandr-devel
- libxdg-basedir-devel libnotify-devel"
+ libxdg-basedir-devel libnotify-devel wayland-devel wayland-protocols"
```
Comment (ericonr) [inline srcpkgs/dunst/template:9]: I'm about 100% sure `wayland-protocols` isn't making any difference here.

Comment (ericonr) [reply srcpkgs/dunst/template:9]: Unless it's being used to determine if Wayland should be built at all.

Comment (FollieHiyuki) [reply srcpkgs/dunst/template:9]: > WAYLAND=(0|1): Disable/Enable wayland support. (Default: 1 (enabled))

I think it always tries to build with Wayland.

Comment (biopsin) [reply srcpkgs/dunst/template:9]: Package wayland-protocols was not found in the pkg-config search path.
Perhaps you should add the directory containing `wayland-protocols.pc'
to the PKG_CONFIG_PATH environment variable
No package 'wayland-protocols' found
Makefile:40: "Failed to query pkg-config for package 'wayland-protocols'!"

Yes enable build default, so you mean you would like wayland be put in a build_option_ ?

**Comment by biopsin** [comment]
@ericonr template updated; when you have the time to take a peek, I'm in a limbo regarding the wayland-protocols comment, build stage complains if it's not pressent (see https://github.com/void-linux/void-packages/pull/29039#discussion_r583447563).

**Comment by ericonr** [comment]
I don't think a build option is necessary, but it's simple enough. And some of their `pkg-config` invocations must require wayland protocols, despite it not being necessary. Thanks for checking :)

**Pull request merged**